### PR TITLE
Add .gitattributes

### PR DESCRIPTION
**Pull request by alexsegura**
| Q             | A
| ------------- | ---
| Bug fix?      | no 
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        | 

Added `.gitattributes`, so that the dist version contains less files

### CODE REVIEW

**Comment by alexsegura** [comment]
Don't understand why PHPStan is failing 🤔 
Should I remove `phpstan.neon.dist` from `.gitattributes`?

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/.circleci export-ignore
+/.github export-ignore
+/features export-ignore
+/tests export-ignore
```
Comment (teohhanhui) [inline .gitattributes]: This is definitely incorrect.

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/.circleci export-ignore
```
Comment (teohhanhui) [inline .gitattributes]: Can we sort everything alphabetically? :smile:

**Comment by teohhanhui** [comment]
`phpstan` failure is unrelated to your changes. Don't worry.

File: .gitattributes (hunk 2 above)
Comment (alexsegura) [reply .gitattributes]: It is indeed sorted alphabetically, in the same way returned by `ls -la` 🤷‍♂️

Comment (teohhanhui) [reply .gitattributes]: `ls -la` sorts directories first. Let's sort alphabetically here.

Comment (alexsegura) [reply .gitattributes]: With dotfiles/dotfolders first, ok for you?

Comment (alexsegura) [reply .gitattributes]: Done ✔️

**Comment by teohhanhui** [comment]
I've rebased this to target the `2.5` branch.

**Comment by teohhanhui** [comment]
Thanks @alexsegura! :tada:

**Pull request merged**